src/libs/brand-utils.js

```diff
@@ -0,0 +1,34 @@
+import { brands } from 'src/libs/brands'
+import { getConfig } from 'src/libs/config'
+import * as Utils from 'src/libs/utils'
+
+
+export const isAnvil = () => (window.location.hostname === brands.anvil.hostName) || getConfig().isAnvil
+export const isBaseline = () => (window.location.hostname === brands.baseline.hostName) || getConfig().isBaseline
+export const isBioDataCatalyst = () => (window.location.hostname === brands.bioDataCatalyst.hostName) || getConfig().isBioDataCatalyst
+// TODO: Deprecate Datastage (https://broadworkbench.atlassian.net/browse/SATURN-1414)
+export const isDatastage = () => (window.location.hostname === brands.datastage.hostName) || getConfig().isDatastage
+export const isElwazi = () => (window.location.hostname === brands.elwazi.hostName) || getConfig().isElwazi
+export const isFirecloud = () => (window.location.hostname === brands.firecloud.hostName) || getConfig().isFirecloud
+export const isProjectSingular = () => (window.location.hostname === brands.projectSingular.hostName) || getConfig().isProjectSingular
+export const isRareX = () => (window.location.hostname === brands.rareX.hostName) || getConfig().isRareX
+export const isTerra = () => (window.location.hostname === brands.terra.hostName) || getConfig().isTerra ||
+  (!isFirecloud() && !isDatastage() && !isAnvil() && !isBioDataCatalyst() && !isBaseline() && !isElwazi() && !isProjectSingular() && !isRareX())
+
+export const getEnabledBrand = () => Utils.cond(
```

Comment:
To reduce some repetitive code, I think I would propose swapping the `isBrand` and `getEnabledBrand` functions. Maybe something like:

```js
export const getEnabledBrand = () => _.find(brand => brand.hostname ==== window.location.hostname || getConfig().brand === brand.name, brands) || brands.terra

export const isAnvil = () => getEnabledBrand() === brands.anvil
```

Reply:
I like the idea. However, `getConfig()` doesn't have a defined `brand` property.

My goal is to further refactor the code around this and I think your comment will be taken care of then.

Reply:
> However, getConfig() doesn't have a defined brand property.

We don't really define any interface for `getConfig` since it returns whatever is in `configOverridesStore`. But yes, this proposal would change the way that the current brand is manually overridden from `configOverridesStore.set({ isAnvil: true })` to `configOverridesStore.set({ brand: 'AnVIL' })`. The current approach allows getting into an invalid state by activating multiple brands at once (`configOverridesStore.set({ isAnvil: true, isBaseline: true })`).

Reply:
You're right. What I have in mind for the follow-up refactoring involves providing a helper (e.g. `enableBrand('baseline')` or perhaps `enableBrand(brands.baseline)`) and hiding the implementation detail (i.e. `configOverridesStore.set({ isBrand: true })`), which should also take care of the issue you raised. 

Since the functional issue you raised is a development feature and is unlikely to happen to a non-developer, I'd like to defer the fix to the follow-up refactoring. Are you okay with that?

Reply:
Sounds good.